app/code/Magento/Catalog/Block/Rss/Category.php

```diff
@@ -257,10 +289,25 @@ public function getFeeds()
     }
 
     /**
-     * {@inheritdoc}
+     * Check isAuthRequired Required
+     *
+     * @return bool
      */
     public function isAuthRequired()
     {
         return false;
     }
+
+    /**
+     * Retrieve config view
+     *
+     * @return \Magento\Framework\Config\View
+     */
+    protected function getConfigView()
```

Comment:
Can `_viewConfig` property of the parent class can be used instead of introducing new dependency?